staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -56,16 +57,28 @@ type DelegatingAuthorizationOptions struct {
 	// AlwaysAllowPaths are HTTP paths which are excluded from authorization. They can be plain
 	// paths or end in * in which case prefix-match is applied. A leading / is optional.
 	AlwaysAllowPaths []string
+
+	// AlwaysAllowGroups are groups which are allowed to take any actions.  In kube, this is system:masters.
+	AlwaysAllowGroups []string
 }
 
 func NewDelegatingAuthorizationOptions() *DelegatingAuthorizationOptions {
 	return &DelegatingAuthorizationOptions{
 		// very low for responsiveness, but high enough to handle storms
 		AllowCacheTTL: 10 * time.Second,
 		DenyCacheTTL:  10 * time.Second,
+
+		// in a kube cluster, system:masters has full power.  Use this as the default which could be later changed.
+		AlwaysAllowGroups: []string{user.SystemPrivilegedGroup},
```

Comment:
I think I'm on board with defaulting to a value that allows coasting, but I expected a flag wired to this to surface it and allow control

Reply:
> I think I'm on board with defaulting to a value that allows coasting, but I expected a flag wired to this to surface it and allow control

This seems like a setting that individual binaries can choose to control how they like.  I think a very small minority (if any) would want to allow an admin to control it and those binaries could choose to wire the value to whatever flag they like.